draft-ietf-quic-http.md

```diff
@@ -1595,9 +1595,11 @@ sent, and receipt MAY be treated as an error of type H3_FRAME_UNEXPECTED.
 # Error Handling {#errors}
 
 QUIC allows the application to abruptly terminate (reset) individual streams or
-the entire connection when an error is encountered.  These are referred to as
-"stream errors" or "connection errors" and are described in more detail in
-{{QUIC-TRANSPORT}}.
+the entire connection; see Sections 2.4 and 5.3 of {{QUIC-TRANSPORT}}.  This
+is referred to as a "stream error" or "connection error" (see
+Section 11 of {{QUIC-TRANSPORT}}) and has an associated error code, but does not
+necessarily have to be an error.  For example, a stream can be reset if the
```

Comment:
This phrasing is strange. What does it mean to not "be an error" when these are referred to as "stream error" or "connection error" ?

Reply:
I agree with you.  My interpretation is that the source of strangeness is calling these stream errors and connection errors in the first place, and calling their reason codes error codes.  This way "error" becomes a technical term defined in this document, which does not align with my mental model of what an error is: entry found is cache or user closing a tab or navigating away is clearly not an error, but the specs treat it is a stream error and assign an error code to it.

Unfortunately I cannot think of a less weird way of phrasing it.  "Do not necessarily have to be abnormal events or protocol violations."?

Reply:
While I agree that it sounds a bit strange, I think the current text is no more confusing as the NO_ERROR error code that we have in the transport draft.